Description: Disable Git repository inspection in Makefile
 Upstream uses Git to determine release information, but this is only valid
 for the upstream Git repository. Disable this makefile logic.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: not-needed
Last-Update: 2020-07-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -195,7 +195,7 @@
 export 	LIBPSM2_COMPAT_CONF_DIR
 
 # The desired version number comes from the most recent tag starting with "v"
-ifeq (true, $(shell git rev-parse --is-inside-work-tree 2>/dev/null))
+ifeq (true, false)
 ISGIT := 1 # Cache the result for later
 # Note, we don't define ISGIT if we are not in a git folder
 VERSION := $(shell git describe --tags --abbrev=0 --match='psm-v*' | sed -e 's/^psm-v//' -e 's/-/_/')
